qiskit_experiments/library/tomography/tomography_analysis.py

```diff
@@ -112,25 +112,37 @@ def _run_analysis(self, experiment_data):
         fitter = self._get_fitter(self.options.fitter)
         fitter_opts = self.options.fitter_options
 
+        # Check for both preparation and measurement data to determine if we are
+        # fitting a channel via QPT or a density matrix via QST
+        qpt = preparation_data.shape[0] and measurement_data.shape[0]
```

Comment:
Why we need to evaluate `measurement_data.shape[0]` too?

Reply:
It's an edge case that doesn't technically apply at the moment unless you manually construct fitter data. In principle if you were to do tomography of preparation without measurement which is possible with the fitters and post processing, it should be reconstructed as a state, not a channel. This doesn't apply to current QPT and QST experiments, but will be relevant for some future ones.